### ISSUE TITLE
get_news() search returning 'NoneType' error

### ISSUE THREAD

**Issue by wcorona269** [Iceloof/GoogleNews#133]
I am getting an error while attempting to use the get_news() function. I have checked the version using gNews.getVersion() and it shows 1.6.11. Half of the time i use the get_news() function it works, half the time it returns a blank list, but every time i get the same 2 errors logged repeatedly.

my code:
```
gNews = GoogleNews(period='20d')
gNews.get_news(topic)

```
The error:
```
 'NoneType' object has no attribute 'parent'
'NoneType' object has no attribute 'startswith'
```

**Comment by samwesley** [comment]
I'm seeing the same issue

**Comment by HurinHu** [comment]
Issue solved with latest version 1.6.12

**Comment by jlchen14** [comment]
Thank you very much for your timely feedback and action~ saved my little project~
